app/models/scopes/template_scope.rb

```diff
@@ -74,7 +74,7 @@ module TemplateScope
     # Retrieves the latest version of each customizable funder template (and the default template)
     scope :latest_customizable, -> {
       family_ids = families(Org.funder.collect(&:id)).distinct.pluck(:family_id) << default.family_id
-      published(family_ids.flatten)
+      published(family_ids.flatten).where('visibility = ? OR is_default = ?', visibilities[:publicly_visible], true)
```

Comment:
flatten is not needed at all since family_ids is already a unidimensional array. The where clause for visibility seems fine however don't see the need for adding again is_default condition...